docs/cloud/connecting.md

```diff
@@ -126,7 +126,7 @@ Use the `prefect config view` CLI command to make sure your execution environmen
 
 <div class="terminal">
 ```bash
-$ prefect config view
+prefect config view
```

Comment:
consider splitting the command and output into separate code blocks, since without the `$`, it's less clear what the user inputs vs what they should expect to see

For example, this code could be (perhaps - I'm not sure what our terminal style supports):

<pre>
&lt;div class="terminal"&gt;
```bash
prefect config view
```

```
PREFECT_PROFILE='cloud'
PREFECT_API_KEY='pnu_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx' (from profile)
PREFECT_API_URL='https://api-beta.prefect.io/api/accounts/...' (from profile)
```
&lt;/div&gt;
</pre>